### ISSUE TITLE
Output log con información de epochs y tamaño sets

### ISSUE THREAD

**Issue by geru-scotland** [geru-scotland/lore-nexus#25]
Estaría genial crear un `output` log en el que se muestre el progreso de las épocas, el training loss - val loss, y también, incluir en el run `.log `actual información sobre el tamaño de los datasets utilizados, número de instancias cada uno.
